Doc/library/functools.rst

```diff
@@ -275,8 +275,8 @@ The :mod:`functools` module defines the following functions:
    .. versionchanged:: 3.8
       Added the *user_function* option.
 
-   .. versionadded:: 3.9
-      Added the function :func:`cache_parameters`
+   .. versionchanged:: 3.9
+      Added the function :func:`!cache_parameters`
```

Comment:
It sounds to me based on this writing that this function was added to the module (which, as far as I can tell, it wasn't), as opposed to being an additional behavior in `cmp_to_key`. Assuming the latter is in fact the case, and since its non-trivial to find more information about it (not being crosslinked), I'd rewrite this as something like:

```suggestion
      Instrument the wrapped callable with a :func:`!cache_parameters` function
      to retrieve the *maxsize* and *typed* of its cache.
```

Reply:
I agree, but I deliberately did not change it. It may be beneficial to address this in a separate PR.